controlplane/pkg/apis/local/connection/connection.proto

```diff
@@ -47,7 +47,7 @@ message Connection {
     string id = 1;
     string network_service = 2;
     Mechanism mechanism = 3;
-    connectioncontext.ConnectionContext context = 4;
+    connectioncontext.IPContext context = 4;
```

Comment:
That is strange. Iwoudl very much vote for keeping the "ConnectionCOntext" here, as it name implies it will keep the context associated with this connection. Please consider reverting this and the related changes.

Reply:
Yep, I planning to use `ConnectionContext`. It's just for testing.

Reply:
+1 on making IPContext a subfield of ConnectionContext :)

Reply:
done